android/app/src/main/AndroidManifest.xml

```diff
@@ -31,14 +31,14 @@
                 <category android:name="android.intent.category.LAUNCHER" />
 
             </intent-filter>
-            <intent-filter>
+            <intent-filter android:autoVerify="true">
                 <action android:name="android.intent.action.VIEW" />
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="helium" />
```

Comment:
We dont need this scheme anymore?

Reply:
We do, but it's in a different intent-filter